### ISSUE TITLE
Codegen not working in V3.9.0

### ISSUE THREAD

**Issue by SebSchwartz** [aws-amplify/amplify-cli#2404]
**Describe the bug**
When adding codegen, it brokes all amplify-meta.json files.

**To Reproduce**
1. Just amplify codegen add --apiId XXXX
```terminal
$ sudo amplify codegen add --apiId XXX
✔ Getting API details
✖ Getting API details
Unexpected token o in JSON at position 1
```
-> amplify-meta.json in both directories (current & backend) now looks like: [object Object]

**Expected behavior**
Possibility to add codegen with no error.

**Desktop (please complete the following information):**
CLI 3.9.0

**Comment by SebSchwartz** [comment]
I'm using a profile with amplify, don't know if this could help debug.

Using the CLI, i'm able to do: aws appsync get-graphql-api --api-id XXX --region eu-west-1

**Comment by nikhname** [comment]
This is a duplicate of #2360. The fix has been merged and released version 3.10.0

**Comment by SebSchwartz** [comment]
Working great again, thanks.
